app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="crayons-layout__sidebar-left">
+  <p class="fw-heavy fs-3xl" style="margin-bottom: 29px;">Mod Center</p>
+  <div id="mod-center-inbox-toggle" style="display: flex; justify-content: space-between; padding: 8px;">
+    <span class="fw-medium">Inbox</span>
+    <span>*</span>
```

Comment:
Hmm I actually don't see this in the specs, what's this for?

Reply:
Now that ModPanel is merged in, I have access to the chevron svg

This refers to this design in the specs:
<img width="593" alt="Screen Shot 2020-06-09 at 3 13 55 PM" src="https://user-images.githubusercontent.com/32520970/84189823-e5ac9b00-aa63-11ea-9b26-80e4652da513.png">

https://www.figma.com/file/Jgq2Qg3MqmjQ4uVdJtyAsS/Moderation?node-id=419%3A6974